plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/Errors/ErrorMessages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\StoreApi\Routes\V1\Agentic\Errors;
+
+/**
+ * Class ErrorMessages
+ *
+ * Manages error messages for the agentic checkout process.
+ */
+class ErrorMessages {
```

Comment:
We can address this later. 

Should be `Messages` only as based on the specs, there are two types of Messages `info` and `error`. 

```suggestion
class Messages {
```